app/controllers/bookings_controller.rb

```diff
@@ -1,6 +1,8 @@
 class BookingsController < ApplicationController
   def index
-    @bookings = Booking.all.where('user = current_user')
+    @bks = Booking.all.where('user = current_user')
+    @bookings = @bks.select {|booking| booking.start_time.utc > Date.today}
+    @new_bookings = @bks.reject {|booking| booking.start_time.utc > Date.today}
```

Comment:
Could you be more specific with the terminology?

Reply:
no